.rspec

```diff
@@ -1 +1 @@
---require spec_helper
+--require rails_helper
```

Comment:
I think this was fine, we should require `spec_helper` here. Why was the change?

Reply:
`rails_helper` includes `spec_helper`, and allows me to remove `require 'rails_helper'` from `spec` files.

Reply:
https://github.com/rspec/rspec-rails/issues/1180